<launch>
        <rosparam param="sync_delta">0.015</rosparam>
        <rosparam param="ir_topic">"/flir_boson/image_rect"</rosparam>
        <rosparam param="rgb_topic">"/camera/color/image_rect_color"</rosparam>
        <rosparam param="odom_topic">"/odom"</rosparam>
        <node pkg="image_process" name="image_process" type="image_process.py"  output="screen" cwd="node"/>
</launch>
